website/www/site/content/en/documentation/runtime/environments.md

```diff
@@ -87,77 +202,28 @@ python -m apache_beam.examples.wordcount \
-By default, no licenses/notices are added to the docker images.
-
-To examine the containers that you built, run `docker images` from anywhere in the command line. If you successfully built all of the container images, the command prints a table like the following:
-```
-REPOSITORY                         TAG                 IMAGE ID            CREATED           SIZE
-apache/beam_java8_sdk              latest              ...                 2 weeks ago       ...
-apache/beam_java11_sdk             latest              ...                 2 weeks ago       ...
-apache/beam_python2.7_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.5_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.6_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_python3.7_sdk          latest              ...                 2 weeks ago       ...
-apache/beam_go_sdk                 latest              ...                 2 weeks ago       ...
-```
-
-### Overriding default Docker targets
-
-The default [tag](https://docs.docker.com/engine/reference/commandline/tag/) is sdk_version defined at [gradle.properties](https://github.com/apache/beam/blob/master/gradle.properties) and the default repositories are in the Docker Hub `apache` namespace.
-The `docker` command-line tool implicitly [pushes container images](#pushing-container-images) to this location.
+{{< highlight class="runner-dataflow" >}}
+export IMAGE="my-repo/beam_python_sdk_custom"
+export TAG="X.Y.Z"
 
-To tag a local image, set the `docker-tag` option when building the container. The following command tags a Python SDK image with a date.
-```
-./gradlew :sdks:python:container:py36:docker -Pdocker-tag=2019-10-04
-```
-
-To change the repository, set the `docker-repository-root` option to a new location. The following command sets the `docker-repository-root`
-to a repository named `example-repo` on Docker Hub.
-```
-./gradlew :sdks:python:container:py36:docker -Pdocker-repository-root=example-repo
-```
+export GCS_PATH="gs://my-gcs-bucket"
+export GCP_PROJECT="my-gcp-project"
+export REGION="us-central1"
 
-## Pushing container images
-
-After [building a container image](#building-container-images), you can store it in a remote Docker repository.
-
-The following steps push a Python3.6 SDK image to the [`docker-root-repository` value](#overriding-default-docker-targets).
-Please log in to the destination repository as needed.
-
-Upload it to the remote repository:
-```
-docker push example-repo/beam_python3.6_sdk
-```
-
-To download the image again, run `docker pull`:
-```
-docker pull example-repo/beam_python3.6_sdk
-```
+# Run a pipeline on Dataflow.
+# This is a Python batch pipeline, so to run on Dataflow Runner V2
+# you must specify the experiment "use_runner_v2"
```

Comment:
Please put this into a code block or reformat.

Reply:
The {{< highlight class="runner-X" >}} formats it into code-blocks, tabbed by runner type - see existing page https://beam.apache.org/documentation/runtime/environments/#testing-customized-images

Reply:
Got it. I went to 'view file' in GitHub and it only showed the rendered markdown which made each of these H1 titles. Is there a way to preview?

Reply:
The best way is to serve the website from source: `./gradlew :website:serveWebsite`

Reply:
You can also view the details of Website_Stage_GCS ("Run Website_Stage_GCS PreCommit") to find the staged version of the website. You can rerun the test if the link's gone stale.